tests/integration/defs/accuracy/test_llm_api_pytorch.py

```
@@ -2576,6 +2576,47 @@ def test_nvfp4_4gpus(self, fp8kv, attention_dp, cuda_graph,
             task = GSM8K(self.MODEL_NAME)
             task.evaluate(llm)
 
+    @skip_pre_rubin
```

Ruff (0.16.2)

[error] Undefined name `skip_pre_rubin`

(F821)